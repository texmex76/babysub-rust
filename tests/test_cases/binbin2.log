c BabySub Subsumption Preprocessor
c reading from '/home/gstrein/Documents/repos/babysub-rust/tests/test_cases/binbin2.cnf'
c parsed 'p cnf 2 3' header
c checked:                      1    clauses 1.00 per subsumed
c subsumed:                     1    clauses 33%
c process-time:                 0.00 seconds
