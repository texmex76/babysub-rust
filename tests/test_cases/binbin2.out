p cnf 2 2
c hash-signature: 13752085431583871717
-1 2 0
1 2 0
